stdlib/2and3/filecmp.pyi

```diff
@@ -2,16 +2,29 @@
 import sys
 from typing import AnyStr, Callable, Dict, Generic, Iterable, List, Optional, Sequence, Tuple, Union, Text
 
+if sys.version_info >= (3, 6):
+    from pathlib import Path
```

Comment:
Should be `os.PathLike`, so that all PathLike objects are supported, not just `pathlib.Path`.